pyproject.toml

```diff
@@ -1,10 +1,89 @@
 [build-system]
 requires = [
-  "setuptools",
-  "setuptools_scm",
-  "wheel",
+  "setuptools>=62.1",
+  "setuptools_scm[toml]>=6.2",
+  "wheel",]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ndcube"
+description = "A package for multi-dimensional contiguous and non-contiguous coordinate aware arrays."
+requires-python = ">=3.9"
+readme = { file = "README.rst", content-type = "text/x-rst" }
+license = { file = "licenses/LICENSE.rst", content-type = "text/plain" }
+authors = [
+  { name = "The SunPy Community", email = "sunpy@googlegroups.com" },
+]
+dependencies = []
```

Comment:
We should have some dependencies here no? :stuck_out_tongue:

Reply:
I've pulled some in, unsure whether thats the complete list or not